core-rust/state-manager/Cargo.toml

```diff
@@ -9,6 +9,7 @@ node-common = { path = "../node-common" }
 sbor = { workspace = true }
 radix-transactions = { workspace = true }
 radix-transaction-scenarios = { workspace = true }
+radix-clis = { workspace = true }
```

Comment:
Any reason for including `radix-clis`?

Reply:
There are some structures there which `ledger-tools` requires